pages/mesosphere/dcos/services/data-science-engine/2.1.0/quick-start/index.md

```diff
@@ -0,0 +1,129 @@
+- Depending on your security mode, {{ model.techName }} requires service authentication for access to DC/OS. See [Provisioning a service account](/mesosphere/dcos/services/data-science-engine/2.1.0/security/#provisioning-a-service-account) for more information.
+
+<p class="message--important"><strong>NOTE: </strong>The following points should be considered before installing the service:
+<br />
+1. {{ model.techName }} uses WebSockets to drive the console, so load balancers and proxies must be configured accordingly.<br />
+2. AWS Classic Load Balancer should not be used, because it does not support WebSockets.
+<br />
+3. By default, the DC/OS Universal Installer uses Classic ELBs (Elastic Load Balancers) for both external and internal load balancing until version 2.0.</p>
+
+<p class="message--note"><strong>NOTE: </strong> If you are planning to use <a href="https://docs.d2iq.com/mesosphere/dcos/services/data-science-engine/2.1.0/storage/hdfs/">HDFS</a> on {{ model.techName }}, you will need a minimum of five nodes.</p>
+
+| Security Mode | Service Account |
+|----------------|------------------|
+| Disabled | Not available |
+| Permissive | Optional |
+| Strict | Required |
+
+# Install {{ model.techName }}
+
+## From the DC/OS UI
+
+1. Select the **Catalog** tab, and search for {{ model.techName }}. Select the {{ model.packageName }} package.
+
+2. Choose the **Review & Run** button to display the **Edit Configuration** page.
+
+3. Configure the package settings using the DC/OS UI or by choosing **JSON Editor** and modifying the app definition manually. For example, you might customize the package by enabling HDFS support.
+
+4. Click **Review & Run**.
+
+5. Review the installation notes, then click **Run Service** to deploy the {{ model.packageName }} package.
+
+
+## From the command line
+
+Install the {{ model.packageName }} package. This may take a few minutes. This step installs the {{ model.packageName }} service.
+
+   ```bash
+   dcos package install -y {{ model.packageName }}
+   ```
+
+   Expected output:
+
+   ```bash
+    Installing Marathon app for package [data-science-engine] version [2.1.0]
+    Installing CLI subcommand for package [data-science-engine] version [2.1.0]
+    New command available: dcos data science-engine
+    DC/OS Data Science Engine is being installed!
+    
+        Documentation: https://docs.mesosphere.com/services/data-science-engine/
+        Issues: https://docs.mesosphere.com/support/
+   ```
+
+
+# Run a Python Notebook Using Spark
+
+1. From DC/OS , select **Services**, then click on the "Open" icon for the {{ model.serviceName }}.
+
+    ![Open JupyterLab](/mesosphere/dcos/services/data-science-engine/img/dcos-jupyter-new-window.png)
+
+    Figure 1 - Open new Jupyter window
+
+    This will open a new window or tab in the browser for JupyterLab.  Log in using the password if specified during the installation of the {{ model.packageName }} package in **Service -> Jupyter Password** option.
```

Comment:
```suggestion
    This will open a new window or tab in the browser for JupyterLab. Log in using the password if specified during the installation of the {{ model.packageName }} package in **Service -> Jupyter Password** option.
```